src/envoy/mixer/utils.cc

```diff
@@ -13,6 +13,7 @@
  * limitations under the License.
  */
 
+#include "common/http/headers.h"
```

Comment:
This should go after `#include "src/envoy/mixer/utils.h"` (also add an empty line), and breaks style check.